tools/trinity.py

```diff
@@ -68,7 +68,7 @@ def post_download(self):
                 with open(badFilePath + '.orig', 'rt') as inf:
                     for line in inf:
                         if line.startswith('unless ($java_version =~ /java version'):
-                            outf.write('$java_version =~ /java version \"1\.(\d+)\./;\n')
+                            outf.write(r'$java_version =~ /java version \"1\.(\d+)\./;\n')
```

Comment:
Ok I've convinced myself that the above three raw strings are fine, but this fourth one in trinity.py still worries me a bit. I've manually deleted the cache container in Travis and re-started the tests on this PR just to give it a full test battery, because I'm pretty sure these particular lines of code don't get tested if Trinity is already present in the build dir's cache.